kafka-ui-react-app/src/components/common/Select/Select.styled.ts

```diff
@@ -81,6 +82,10 @@ export const OptionList = styled.ul`
     background-color: ${(props) =>
       props.theme.select.optionList.scrollbar.backgroundColor};
   }
+
+  &::-webkit-scrollbar:horizontal {
+    display: none;
```

Comment:
I'm not sure about hiding it 
I think we either just have to add our styles or add `-webkit-appearance: auto;` (it will show only when user scrolls and it has scroll)

Reply:
I have to add - that i don't think select has to scroll horizontally but i think i will be great indicator that we done something wrong

Reply:
<img width="813" alt="Screenshot 2022-02-04 at 12 28 02" src="https://user-images.githubusercontent.com/83412197/152505105-122b02ba-a012-44ca-a8cb-c84d6fc470c0.png">

Maybe I just lower the height?

Reply:
Looks good to me